src/turn.erl

```diff
@@ -592,11 +586,29 @@ send_payload_to_parent(State, Payload) ->
                 {ice_controlled, StunMsg#stun.'ICE-CONTROLLED'},
                 {ice_controlling, StunMsg#stun.'ICE-CONTROLLING'}],
                self()};
-        false ->
-            NewState#state.parent ! {ice_payload, Payload}
+        {Parent, false} ->
+            Parent ! {ice_payload, Payload}
     end,
     NewState.
 
+try_resolve_parent(#state{parent = undefined} = State) ->
+    {_Addr, Port} = State#state.candidate_addr,
+
+    case lists:member(Port, State#state.port_black_list) of
+        true ->
+            State;
+        false ->
+            case (State#state.parent_resolver)(Port) of
+                {ok, Parent} ->
+                    State#state{parent = Parent};
+                {error, _Reason} ->
+                    NewPortBlackList = [Port] ++ State#state.port_black_list,
+                    State#state{port_black_list = NewPortBlackList}
+            end
```

Comment:
I think we should log this fact e.g. "CC for unknown port" or something like this